cache-tests.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var Cache = require('../cache');
```

Comment:
This points outside the repo - is my git showing something different from yours?